config/prod.secret.exs

```diff
@@ -0,0 +1,22 @@
+use Mix.Config
+
+# In this file, we keep production configuration that
+# you'll likely want to automate and keep away from
+# your version control system.
+#
+# You should document the content of this
+# file or create a script for recreating it, since it's
+# kept out of version control and might be hard to recover
+# or recreate for your teammates (or yourself later on).
+config :omscore, OmscoreWeb.Endpoint,
+  secret_key_base: "UuUfhhDxYJ0wg6Drz4FYvrC+P36lLkoan4f4cW0LAtnPcpD8FVf6DKpfMy3ek09e"
```

Comment:
@blacksph3re is this safe?

Reply:
Yes, it is used for http browser sessions in elixir, and we don't use any sessions. I could try removing it though.